crates/sui-graphql-rpc/src/context_data/pg_backend.rs

```diff
@@ -631,16 +632,38 @@ impl PgQueryExecutor for PgManager {
         self.inner
             .spawn_blocking(move |this| {
                 let query = query_builder_fn()?;
-                let explain_result: String = this
-                    .run_query(|conn| query.explain().get_result(conn))
-                    .map_err(|e| Error::Internal(e.to_string()))?;
-                let cost = extract_cost(&explain_result)?;
-                if cost > max_db_query_cost as f64 {
-                    return Err(DbValidationError::QueryCostExceeded(
-                        cost as u64,
-                        max_db_query_cost,
-                    )
-                    .into());
+                let explain_result: Option<String> =
+                    match this.run_query(|conn| query.explain().get_result(conn)) {
+                        Ok(result) => Some(result),
+                        Err(e) => {
+                            info!(
+                                target: "async-graphql",
```

Comment:
I would recommend using a different target for this, so that we can control its log level separately.